core/templates/App.ts

```diff
@@ -79,6 +79,24 @@ angular.module('oppia').config([
   function(
       $compileProvider, $cookiesProvider, $httpProvider,
       $interpolateProvider, $locationProvider, $provide) {
+    var ugs = new UpgradedServices();
+    // We need to provide these services separately since they are
```

Comment:
Is there really a benefit in removing UpgradedServices, then? This makes me think it's not actually worth the effort?

Reply:
We are not removing the UpgradedServices but the PR description now covers why we are doing this. Please let me know if it's still unclear :).